src/Tribe/Commerce/Tickets_Commerce/Settings.php

```diff
@@ -0,0 +1,359 @@
+<?php
+
+namespace Tribe\Tickets\Commerce\Tickets_Commerce;
+
+use Tribe\Tickets\Commerce\Tickets_Commerce\Gateways\Abstract_Gateway;
+use Tribe__Field_Conditional;
+
+/**
+ * The Tickets Commerce settings.
+ *
+ * This class will contain all of the settings handling and admin settings config implementation from
+ * Tribe__Tickets__Commerce__PayPal__Main that is gateway-agnostic.
+ *
+ * @since   TBD
+ * @package Tribe\Tickets\Commerce\Tickets_Commerce
+ */
+class Settings extends Abstract_Settings {
+
+	/**
+	 * The option key for enable.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_enable = 'tickets-commerce-enable';
+
+	/**
+	 * The option key for sandbox.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_sandbox = 'ticket-paypal-sandbox';
+
+	/**
+	 * The option key for currency code.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_currency_code = 'ticket-paypal-currency-code';
+
+	/**
+	 * The option key for stock handling.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_stock_handling = 'ticket-paypal-stock-handling';
+
+	/**
+	 * The option key for success page.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_success_page = 'ticket-paypal-success-page';
+
+	/**
+	 * The option key for confirmation email sender email.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_confirmation_email_sender_email = 'ticket-paypal-confirmation-email-sender-email';
+
+	/**
+	 * The option key for confirmation email sender name.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_confirmation_email_sender_name = 'ticket-paypal-confirmation-email-sender-name';
+
+	/**
+	 * The option key for confirmation email subject.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $option_confirmation_email_subject = 'ticket-paypal-confirmation-email-subject';
+
+	/**
+	 * Get the list of settings for Tickets Commerce.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The list of settings for Tickets Commerce.
+	 */
+	public function get_settings() {
+		$home_url = home_url();
+
+		$plus_link    = sprintf(
+			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
+			esc_html__( 'Event Tickets Plus', 'tribe-common' )
+		);
+		$plus_link_2  = sprintf(
+			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
+			esc_html__( 'Check it out!', 'tribe-common' )
```

Comment:
```suggestion
			esc_html__( 'Check it out!', 'event-tickets' )
```

Reply:
Missing translator comments.

Reply:
No translator comment needed here since the placeholder isn't in an `__()`